app/services/analytics_events.rb

```diff
@@ -1369,12 +1363,12 @@ def idv_doc_auth_document_capture_polling_wait_visited(
   # @param [Boolean] opted_in_to_in_person_proofing User opted into in person proofing
   def idv_doc_auth_document_capture_submitted(
     success:,
-    errors:,
     step:,
     analytics_id:,
     liveness_checking_required:,
     selfie_check_required:,
     flow_path:,
+    errors: nil,
```

Comment:
[question] curious about the couple of events still using `events`. should we not migrate those other? is there a worry it will break historic queries or something?

Reply:
I think largely yes, though there's a lot of varied inconsistency in what remains after this. Many are one-off event logging passing `events` directly rather than using `FormResponse#to_h` ([example](https://github.com/18F/identity-idp/blob/064352bd8db3b3901edc71598b4a898f173e3af0/app/controllers/users/webauthn_setup_controller.rb#L50)). There's also a handful of cases where `FormResponse` is initialized using a hash instead of `ActiveModel::Errors`. I think it'd be a good idea to migrate to using the latter consistently, though I'd expect those would require some non-trivial refactoring in each individual instance.